install-sddm-theme.sh

```
@@ -242,6 +266,14 @@ function skipLoadingAnimation {
   done
 }
 
+function compileShaders {
+    if [ ! -f components/doorShader.frag.qsb ]; then
+        echo "Compiling shaders..."
+        cd components
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)